_posts/2021-12-10-start-a-workspace-by-entering-a-url.adoc

```diff
@@ -0,0 +1,126 @@
+====
+`#https://github.com/eclipse-che/che-plugin-registry/pull/1097`
+====
+
+URL syntax for cloning GitLab-hosted repositories:
+
+* To clone the default branch on GitLab to the starting workspace:
++
+`#https://gitlab.com/_<user_or_org>_/_<repository>_`
+
+* To clone a specified branch on GitLab to the starting workspace:
++
+`#https://gitlab.com/_<user_or_org>_/_<repository>_/-/tree/_<branch>_`
+
+
+=== Using a different editor or Integrated Development Environment (IDE)
+
+{prod} starts workspaces with a default IDE: {prod-short} Theia.
+
+Overriding the default editor is possible with the `?che-editor=__<editor-id>__` parameter. The `editor-id` comes from the identifier of the IDE in the plug-in registry.
+
+.Using the next version of Theia (build after each commit) 
+====
+`pass:c,a,q[{prod-url}#https://github.com/example/foo?che-editor=eclipse/che-theia/next]`
+====
+
+Some available IDE keys:
+
+* `eclipse/che-theia/latest`
+* `eclipse/che-theia/next`
+* `che-incubator/intellij-community/latest`
+* `che-incubator/pycharm/latest` 
+* `che-incubator/che-code/insiders`
+
+The parameter `?che-editor` accepts a link to the devfile hosted on a public server (with Cross-Origin Resource Sharing (CORS) enabled).
+
+====
+`pass:c,a,q[{prod-url}#https://github.com/example/foo?che-editor=https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/che-incubator/che-code/insiders/devfile.yaml]`
+====
+
+=== Using a different devfile file from the repository
+
+If the devfile file name is different from the default search pattern, which is `.devfile.yaml` or `devfile.yaml`, it is possible to override that name.
+
+Use the parameter `?devfilePath=mydevfile.yaml` at the end of `{prod-url}#__<git_repository_url>__`.
+
+====
+`pass:c,a,q[{prod-url}#https://github.com/example/foo?devfilePath=mydevfile.yaml]`
+====
+
+The short name for the `devfilePath` parameter is `df`.
+
+====
+`pass:c,a,q[{prod-url}#https://github.com/example/foo?df=mydevfile.yaml]`
+====
+
+
+== URL syntax for workspace names
+
+
+If you enter a URL such as `pass:c,a,q[{prod-url}#http://github.com/foo/bar]` and you don’t have any workspace named `bar`, {prod} creates a workspace named `bar` and clones the GitHub repository.
+
+Each time you enter a URL such as `{prod-url}#http://github.com/foo/bar` again, {prod} reopens the existing workspace `bar`.
+
+To create another workspace each time you enter the same URL, enter the URL appended with `?new` in the URL format `pass:c,a,q[{prod-url}#http://github.com/foo/bar?new]`. Even if you already have a workspace named `bar`, each time you enter such URL, {prod} will create another, sequentially named workspace (example: `bar-abcde`).
+
+=== Support of devfiles
+
+{prod} uses {devworkspace} Operator that targets devfiles v2 with `schemaVersion: 2.1.0`.
+Use of devfile v1 is deprecated. Under the hood, the https://github.com/che-incubator/devfile-converter[converter library] converts devfiles v1 to devfiles v2.
```

Comment:
> converter library converts

to avoid tautology maybe it is better to drop `converter` or replace `converts` with a synonym e.g. transforms

Reply:
updated 👍